crates/project-model/src/sysroot.rs

```diff
@@ -34,14 +34,14 @@ pub struct Sysroot {
 }
 
 #[derive(Debug, Clone, Eq, PartialEq)]
-pub(crate) enum SysrootWorkspace {
+pub enum SysrootWorkspace {
```

Comment:
While we're here, should this be renamed to `RustSrc`, since this isn't the sysroot _per-se_, but rather a component in a sysroot called `rust-src`?